.github/workflows/build.yml

```diff
@@ -22,10 +22,10 @@ jobs:
       GOARCH: amd64
       GOPROXY: https://proxy.golang.org
       JDK_VER: 13.0.x
-      DAPR_RUNTIME_VER: 0.9.0-rc.1
-      DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/e498de9f7dd92c30aa592d6e6761dc924bb53cc2/install/install.sh
-      DAPR_CLI_REF: 35b9a824d2fd9e2dcf8e75d1d49ce59a39c7cf5b
-      DAPR_REF: b53586d1e5a880a3f87044975ace25b9ae51daec
+      DAPR_RUNTIME_VER: 0.10.0-rc.0
+      DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/e7c9a643dfefbcfff0c2c26c12029259e6e81180/install/install.sh
+      DAPR_CLI_REF: e7c9a643dfefbcfff0c2c26c12029259e6e81180
+      DAPR_REF: 98365b1b9ade55e7cf46cbc2313f0625318c0977
```

Comment:
i think this Dapr Ref is not needed as it refers to the same runtime version

Reply:
We can fix this when we cut the release today.